News updates and man page fixes

daviesrob opened the pull request:
Updated NEWS for samtools.  Hopefully nothing important has been left out.

Includes some man page updates - `sam.gz` output and indexing; and the `samtools fasta` default value for the `-F` option.

jmarshall commented on NEWS:
```diff
@@ -1,79 +1,275 @@
+
+ * samtools now supports alignment records with reference positions greater
+   than 2 gigabases.  This allows samtools to process alignments for
+   species which large chromosomes like axolotl and lungfish.  Note that
```
```suggestion
   species which have large chromosomes, like axolotl and lungfish.  Note that
```

jmarshall commented on NEWS:
```diff
@@ -1,79 +1,275 @@
+   This should work with addreplacerg, depad, markdup, merge, split,
+   and view. (#1062)
+
+ * A global '--vebosity' option has been added to enable/disable
```
```suggestion
 * A global '--verbosity' option has been added to enable/disable
```

jmarshall commented on NEWS:
```diff
@@ -1,79 +1,275 @@
+ * HTSlib (and therefore SAMtools) no longer considers a zero-length
+   file to be a valid SAM file.  This has been changed so that pipelines such
+   as `somecmd | samtools ...` with `somecmd` aborting before outputting
+   anything will not propagate the error to the second command.
```
Same typo copied over from the other one! :smile:

```suggestion
   anything will propagate the error to the second command.
```

jmarshall commented on NEWS:
```diff
@@ -1,79 +1,275 @@
+ * Fixed incorrect macro definition on Windows. (#950)
+
+ * bedcov, phase, misc/ace2sam and misc/wgsim now check for failure to open
+   files.  (#1013, thanks to John Marshall)
```
Also thanks to the person who reported it

```suggestion
   files.  (#1013, thanks to Julie Blommaert and John Marshall)
```

jmarshall commented on NEWS:
Just `(#1073, #1100)`; my contribution here was minimal.

daviesrob commented:
Updated, along with removal of some stray trailing white-space.  I removed the YAML bit from markdup stats output as we (myself and @whitwham) weren't entirely sure that it was true.
